pkg/universe.dagger.io/x/vgjm456@qq.com/helm/helm.cue

```diff
@@ -0,0 +1,93 @@
+package helm
+
+import (
+	"dagger.io/dagger"
+	// "dagger.io/dagger/core"
+
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/bash"
+	// "universe.dagger.io/docker"
+)
+
+#Install: {
+	// Name of your release
+	name:       string | *""
+	kubeconfig: dagger.#Secret
+
+	source: *"repository" | "URL" | "local"
+
+	if source == "repository" {
+		chart:      string
+		repository: string
+	}
+
+	if source == "URL" {
+		URL: string
+	}
+
+	// -----
+
+	if source == "repository" {
+		chart:      string
+		repository: string
+		env: {
+			REPOSITORY: repository
+			CHART:      chart
+		}
+		script: contents: #"""
+				helm repo add somerepo $REPOSITORY
+				helm repo update
+				helm install \#(name) somerepo/$CHART \#(_generateName)
+				"""#
+	}
+
+	if source == "URL" {
+		URL: string
+		run: script: contents: #"""
+				helm install \#(name) \#(URL) \#(_generateName)
+				"""#
+	}
+
+	run: bash.#Run & {
+		input:  _base.output
+		always: true
+		mounts: "/root/.kube/config": {
+			dest:     "/root/.kube/config"
+			type:     "secret"
+			contents: kubeconfig
+		}
+	}
+
+	_base: #Build
+
+	_generateName: string | *""
+	if name == "" {
+		_generateName: "--generate-name"
+	}
+}
+
+#Build: {
+
+	helmVersion: string | *"v3.8.1"
+
+	kubectlVersion: string | *"v1.23.5"
+
+	bash.#Run & {
```

Comment:
Didn't find a stable version of helm in alpine packages.
Btw, I suppose that alpine.#Build doesn't support adding testing repository packages yet.

Reply:
> We can't use [alpine/helm](https://hub.docker.com/r/alpine/helm) instead of manually install it?

Hi. I just switched to alpine/helm. Maybe you can review again.